composer.json

```diff
@@ -61,7 +61,9 @@
         "steverhoades/oauth2-openid-connect-server": "1.2",
         "nyholm/psr7": "1.3.2",
         "nyholm/psr7-server": "1.0.1",
-        "lcobucci/jwt": "dev-34php8 as 3.4.9"
+        "lcobucci/jwt": "dev-34php8 as 3.4.9",
+        "psr/log": "^1.1",
+        "monolog/monolog": "^2.1"
```

Comment:
the practice we use in composer.json for packges is to pin the  version
(and we generally every month or so then increment them to most recent versions and cross fingers)

Reply:
Fixed this.